### DISCUSSION TITLE
Limiting over Inverter?

### DISCUSSION THREAD

**Discussion by welli2009** [reinhard-brandstaedter/solarflow-control#214] [General]
Hello everyone,

First of all, I would like to thank all the contributors, especially @reinhard-brandstaedter, who have turned the project into what it is now. After some initial difficulties on Hub 1200, things started running smoothly there, and now on the 2000er as well. What's a bit concerning to me is that every few seconds, the INV is being newly limited. I'm not sure if it's related, but the INV is increasingly shutting down and needs to be disconnected from all cables and then reconnected to work again. This may be because the sf control is supplying 0W, rendering it unable to operate. Would it be possible to gradually increase the OutputLimit to ~10W every few seconds so that the INV doesn't shut off?

Thank you for the project. I would also like to contribute, although my Python skills are not very advanced. Perhaps I could go through Typofixes or slt at some point.

Thanks,
Jacob

**Comment by reinhard-brandstaedter** [comment]
Hi @welli2009 ,

We are trying to keep the inverter limit changes to a minimum (rate limiting, only on significant changes) to avoid unnecessary commands on the DTU. Also the inverter itself is never limited below 10W per input channel, which is usually enough to keep it running, if there is power supply.
The only case when the inverter can go down (turn off) is when the solarflow battery is completely empty and thus the hub's limit is set to 0. This is necessary to avoid a increased discharge when it starts up again.
However I haven't seen an inverter having an issue with being cut off power. For inverters that is normal, imagine no sunshine at night (which is quite normal :-) the inverter will also turn off.

Maybe it is not the inverter but the DTU that is not behaving well? What are you using?

kR

**Comment by welli2009** [comment]
For the moment everything seems to work, ig the problem was, that the hub was charging the battery (not full for the last few days). I just set in solarflow.py while charging the output to 50W. I’m using an openDTU which is placed 5m away from the INV, but it was completely shut down, the status LED was turned off.
   
  
   thx in advance,
   
  
   Jacob
